src/Actions/Actions/PrivacyExtensions.cs

```diff
@@ -1,103 +1,19 @@
-
-        internal static A11yElement GetScrubbedElementTree(this A11yElement pointOfInterest)
-        {
-            A11yElement previous = null;
-            for (A11yElement currentNode = pointOfInterest; currentNode != null; currentNode = currentNode.Parent)
-            {
-                A11yElement clone = currentNode.CreateScrubbedCopyWithoutRelationships();
-
-                if (previous != null)
-                {
-                    clone.Children.Add(previous);
-                    previous.Parent = clone;
-                }
-
-                previous = clone;
-            }
-
-            return previous;
-        }
-
-        /// <summary>
-        /// Given the input element, create a scrubbed copy that has no parent or children
-        /// </summary>
-        /// <param name="element">The element to clone</param>
-        /// <returns>A new element that has been scrubbed of key information and properties</returns>
-        internal static A11yElement CreateScrubbedCopyWithoutRelationships(this A11yElement element)
-        {
-            A11yElement clone = new A11yElement
-            {
-                IssueDisplayText = element.IssueDisplayText,
-                ScanResults = element.ScanResults,
-                TreeWalkerMode = element.TreeWalkerMode,
-                UniqueId = element.UniqueId,
-                Children = element.Children == null ? null : new List<A11yElement>()
-            };
-
-            if (element.Properties != null)
-            {
-                clone.Properties = new Dictionary<int, A11yProperty>();
-                foreach (var pair in element.Properties)
-                {
-                    if (PropertiesToInclude.Contains(pair.Key))
-                    {
-                        if (pair.Key == PropertyType.UIA_NamePropertyId)
-                        {
-                            A11yProperty scrubbedProperty = new A11yProperty(pair.Key, pair.Value.ToString().Substring(0, MaxScrubbedNameLength));
-                            clone.Properties.Add(pair.Key, scrubbedProperty);
-                        }
-                        else
-                        {
-                            clone.Properties.Add(pair.Key, pair.Value);
-                        }
-                    }
-                }
-            }
-
-            if (element.Glimpse != null)
-            {
-                clone.UpdateGlimpse();
-            }
-
-            return clone;
-        }
-
         /// <summary>
         /// Given the root to an element tree, synthesize a "yellow box" bitmap to describe it
         /// </summary>
         /// <param name="rootElement">The root element for the tree</param>
         /// <returns>A bitmap representing the element tee (normalized to having the app window at location (0,0)</returns>
-        internal static Bitmap SynthesizeBitmapFromElements(this A11yElement rootElement)
+        internal static Bitmap SynthesizeBitmapFromElements(this IA11yElement rootElement)
```

Comment:
>SynthesizeBitmapFromElements [](start = 31, length = 28)

This is used by the Load Action. One could build an argument that it ought to be in AIWin, but that's a much larger change (out of scope for now)